domain/src/main/java/no/difi/meldingsutveksling/arkivmelding/ArkivmeldingFactory.java

```diff
@@ -0,0 +1,162 @@
+package no.difi.meldingsutveksling.arkivmelding;
+
+import lombok.extern.slf4j.Slf4j;
+import no.arkivverket.standarder.noark5.arkivmelding.*;
+import no.arkivverket.standarder.noark5.metadatakatalog.Korrespondanseparttype;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.core.EDUCoreConverter;
+import no.difi.meldingsutveksling.domain.MeldingsUtvekslingRuntimeException;
+import no.difi.meldingsutveksling.nextmove.message.MessagePersister;
+import no.difi.meldingsutveksling.noarkexchange.PutMessageRequestWrapper;
+import no.difi.meldingsutveksling.noarkexchange.schema.core.MeldingType;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.GregorianCalendar;
+import java.util.Optional;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static java.lang.String.format;
+import static java.util.Optional.ofNullable;
+
+@Slf4j
+@Component
+public class ArkivmeldingFactory {
+
+    private IntegrasjonspunktProperties props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "props" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)